examples/packet_forward_test.go

```diff
@@ -0,0 +1,191 @@
+package ibctest_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	transfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	"github.com/strangelove-ventures/ibctest"
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"github.com/strangelove-ventures/ibctest/test"
+	"github.com/strangelove-ventures/ibctest/testreporter"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestPacketForwardMiddleware(t *testing.T) {
```

Comment:
It would be great to get this as a "larger example" as detailed in https://go.dev/blog/examples#larger-examples and https://pkg.go.dev/testing#hdr-Examples.

But examples are expected to be niladic functions, so to get this to run as an example, I think we'd have to provide a mocked `testing.T`, which will end up being a whole can of worms.

Until it's realistic to put that together, why does this call `t.Skip`?

Reply:
Thanks for the links Mark!

I added the `t.Skip` call as a last minute after thought because I wasn't sure if we want all the tests that could live in an `example` directory to run on calls to `go test` since there are intermittent Docker failures that could cause false positives. Happy to remove this line though if you think otherwise.